ambient.d.ts

```diff
@@ -499,14 +541,14 @@ declare module '@mojaloop/central-services-stream' {
     }>
   }
 
-  type ConsumeCallback = <T>(error: Error, payload: T) => Promise<void>;
+  type ConsumeCallback<T> = (error: Error | null, payload: T) => Promise<void>;
```

Comment:
let use more descriptive generic types' names
```suggestion
  type ConsumeCallback<Payload> = (error: Error | null, payload: Payload) => Promise<void>;
```